src/common/filesystembase.cpp

```diff
@@ -578,8 +578,11 @@ bool FileSystem::isChildPathOf(const QString &child, const QString &parent)
     // if it is a relative path assume a local file, resolve it based on root
     const auto sensitivity = Utility::fsCaseSensitivity();
 
-    // Fast-path the 2 common cases in this if-else statement:
```

Comment:
What happened to 2?

Reply:
it became 3: if the parent is empty we return true

Reply:
Yes but shouldn't case 2 still be documented?

Reply:
Done.